gatorgrade/output/output_tools.py

```diff
@@ -1,3 +1,8 @@
-# This file is used for storing supporting functions and tools that are used by the methods in output_functions.
-# The main use of this class is to de-clutter output_functions.py for better readability.
-# One example of a function to be written here is the splitting of strings retrieved from the input team, at spaces, into lists.
\ No newline at end of file
+''' This file is used for storing supporting functions and tools that are used by the methods in output_functions.

```

Comment:
please remove the space before and after the docstring. For example, a docstring should follow this format: """Something here."""